felix/collector/collector.go

```diff
@@ -306,7 +306,13 @@ func (c *collector) getDataAndUpdateEndpoints(t tuple.Tuple, expired bool, packe
 	dstEp := c.lookupEndpoint(t.Src, t.Dst)
 	dstEpIsNotLocal := dstEp == nil || !dstEp.IsLocal
 
-	if !okData {
+	// Ignore HEP reporters.
+	if (srcEp != nil && srcEp.IsHostEndpoint()) ||
```

Comment:
add && srcEp.IsLocal inside the brackets; and similarly && dstEp.IsLocal inside the brackets on the following line.  (Or any refactoring of that logic you prefer).  You can also move this all inside if !exists, which will skip doing the check on non-first updates.

We want to include all flow logs where the reporter side is a pod, so they can have an endpoint (either one) be a hostendpoint